app/examples/qutebrowser

```diff
@@ -0,0 +1,111 @@
+" Options
+set adblocker=update
+set devtoolsposition=split
+set downloadmethod=confirm
+set firefoxmode=always
+set fontsize=16
+set norestoretabs
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set tabclosefocusright
+set tabreopenposition=next
```

Comment:
This can only be set to 'left', 'right' and 'previous'.

Reply:
I will set it to 'right'. I confused it with qutebrowser's next/previous.